### DISCUSSION TITLE
Would love to have charts components for uniform styling with dashboards with charts

### DISCUSSION THREAD

**Discussion by JonathonRP** [huntabyte/shadcn-svelte#570] [Ideas]
There are many charting libraries for svelte and would love to have charts that match with shadcn/ui styling. It's weird they demo a dashboard with chart components but never released any chart components

**Comment by BadisKefi** [comment]
this will be cool but for now, i personally use the charts available in the "ui.shadcn.com/themes"

**Reply by dskvr** [reply]
But those are react charts, no?

**Reply by itsmikesharescode** [reply]
How do you use that in svelte ?

**Comment by JonathonRP** [comment]
Imma be using layerchart for my current side project fyi
